scripts/generate-theme.js

```
@@ -316,13 +316,37 @@ function replacePlaceholders( content ) {
+			if ( filterName ) {
+				switch ( filterName.trim() ) {
+					case 'upper':
+						return value.toUpperCase().replace( /-/g, '_' );
+					case 'snakeCase':
+						return value.toLowerCase().replace( /-/g, '_' );
+					case 'phpCase':
+						return value.toLowerCase().replace( /-/g, '_' );
+					case 'pascalCase':
+						return value.split('-').map(part => part.charAt(0).toUpperCase() + part.slice(1)).join('');
+					case 'camelCase':
+						const parts = value.split('-');
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)